ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,306 @@
+
+    logger "Installing Open Distro for Kibana..."
+    eval "$sys_type install opendistroforelasticsearch-kibana-${OPENDISTRO_VERSION} -y -q $debug"
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Kibana installation failed"
+        exit 1;
+    else
+        eval "curl -so /etc/kibana/kibana.yml https://raw.githubusercontent.com/wazuh/wazuh/${BRANCH}/extensions/kibana/7.x/kibana_all_in_one.yml --max-time 300 $debug"
+        eval "cd /usr/share/kibana $debug"
+
+
+
+        if [ "${STATUS_PACKAGES}" = "prod" ]; then
+          eval "sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/Backups/staging/app/kibana/wazuhapp-${WAZUH_VERSION}_${ELK_VERSION}.zip $debug"
+        elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+          eval "sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/pre-release/ui/kibana/wazuhapp-${WAZUH_VERSION}_${ELK_VERSION}.zip $debug"
+        fi
+
+
+        eval "sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/trash/app/kibana/wazuhapp-4.0.0_7.8.0.zip $debug"
+        if [  "$?" != 0  ]
+        then
+            echo "Error: Wazuh Kibana plugin could not be installed."
+            exit 1;
+        fi
+        eval "mkdir /etc/kibana/certs $debug"
+        eval "mv /etc/elasticsearch/certs/kibana* /etc/kibana/certs/ $debug"
+        eval "setcap 'cap_net_bind_service=+ep' /usr/share/kibana/node/bin/node $debug"
+
+        # Start Kibana
+        startService "kibana"
+
+        logger "Done"
+    fi
+}
+
+## Health check
+healthCheck() {
+
+    cores=$(cat /proc/cpuinfo | grep processor | wc -l)
+    ram_gb=$(free -m | awk '/^Mem:/{print $2}')
+
+    if [[ $cores < "2" ]] || [[ $ram_gb < "4096" ]]
+    then
+        echo "The system must have at least 4Gb of RAM and 2 CPUs"
+        exit 1;
+    else
+        echo "Starting the installation..."
+    fi
+}
+
+checkInstallation() {
+
+    logger "Checking the installation..."
+    eval "curl -XGET https://localhost:9200 -uadmin:admin -k --max-time 300 $debug"
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Elasticsearch was not successfully installed."
+        exit 1;
+    else
+        echo "Elasticsearch installation succeeded."
+    fi
+    eval "filebeat test output $debug"
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Filebeat was not successfully installed."
+        exit 1;
+    else
+        echo "Filebeat installation succeeded."
+    fi
+    logger "Initializing Kibana (this may take a while)"
+    until [[ "$(curl -XGET https://localhost/status -I -uadmin:admin -k -s | grep "200 OK")" ]]; do
+        echo -ne $char
+        sleep 10
+    done
+    echo $'\nInstallation finished'
+    exit 0;
+}
+
+cleanInstall(){
+
+    eval "rm -rf /etc/yum.repos.d/adoptopenjdk.repo"
+    eval "rm -rf /etc/yum.repos.d/opendistroforelasticsearch-artifacts.repo"
+    eval "rm -rf  /etc/yum.repos.d/wazuh.repo"
```

Comment:
Add here `yum clean all` to remove the repository cache data.

Reply:
Done in 528feec208a367ba616fb87ea4237096d0681494